data/hai/hai.txt

```diff
@@ -506,26 +622,33 @@ phrase "hostile unfettered"
 		"Peace"
 		"Peacefulness"
 		"Nonviolence"
-		"Understanding"
-		"Reconciliation"
+		"Humility"
+		"Agreeableness"
```

Comment:
Maybe we replace reconciliation by adding "Easy forgiveness" or something that conveys the idea of forgetting quickly

Also we could add "Giving undeserved trust" or smth to convey that they dont trust everyone by default, not everything has to be in one word?

Also maybe "Lack of Honor"

I just realized how dumbly evil these made the Uhai sound...

Reply:
"Easy forgiveness" is contained within Forbearance.
"Giving undeserved trust" would be either Naivety or Gullibility, but those are also follies of youth and probably not worth being dead-set against.
"Lack of honor" is _probably_ covered by Docility, Passivity and Apathy at least in the ways that the UHai apply it. I think it's a lexical gap for a specific word replacement though. The only other thing here could be "Insincerity", but the Hai and UHai are both pro-sincerity so it's not a point of difference.

Yeah, I only realised how dumbly evil these were when I looked in here for this too and I was like, "lol nope"

Reply:
Well its not because it's already there once we can't have it said in two different ways?
Though it is nice that they be concise with it for sure.